pkg/cmd/root.go

```diff
@@ -101,6 +101,9 @@ func checkVersion(command string) {
 		return
 	}
 
+	wg.Add(1)
+	go controllers.CaptureEvent(wg, "VersionCheck")
```

Comment:
Should these be wrapped with `EnableAnalytics` checks?

Reply:
Good catch. I'm going to move the `IsAnalyticsEnabled` check into the analytics controller to avoid this issue in the future.